src/Locales/fr/Pode.psd1

```diff
@@ -0,0 +1,287 @@
+    encodingAttributeOnlyAppliesToMultipartExceptionMessage           = "L'attribut d'encodage s'applique uniquement aux corps de requête multipart et application/x-www-form-urlencoded."
+    suppliedDateBeforeScheduleStartTimeExceptionMessage               = "La date fournie est antérieure à l'heure de début du calendrier à {0}"
+    unlockSecretRequiredExceptionMessage                              = "Une propriété 'UnlockSecret' est requise lors de l'utilisation de Microsoft.PowerShell.SecretStore"
+    noLogicPassedForMethodRouteExceptionMessage                       = '[{0}] {1}: Aucune logique passée.'
+    bodyParserAlreadyDefinedForContentTypeExceptionMessage            = 'Un analyseur de corps est déjà défini pour le type de contenu {0}.'
+    invalidJwtSuppliedExceptionMessage                                = 'JWT fourni invalide.'
+    sessionsRequiredForFlashMessagesExceptionMessage                  = 'Des sessions sont nécessaires pour utiliser les messages Flash.'
+    semaphoreAlreadyExistsExceptionMessage                            = 'Un sémaphore avec le nom suivant existe déjà: {0}'
+    invalidJwtHeaderAlgorithmSuppliedExceptionMessage                 = "Algorithme de l'en-tête JWT fourni invalide."
+    oauth2ProviderDoesNotSupportPasswordGrantTypeExceptionMessage     = "Le fournisseur OAuth2 ne supporte pas le type de subvention 'password' requis par l'utilisation d'un InnerScheme."
+    invalidAliasFoundExceptionMessage                                 = 'Alias {0} non valide trouvé : {1}'
+    scheduleDoesNotExistExceptionMessage                              = "Le calendrier '{0}' n'existe pas."
+    accessMethodNotExistExceptionMessage                              = "La méthode d'accès n'existe pas : {0}"
+    oauth2ProviderDoesNotSupportCodeResponseTypeExceptionMessage      = "Le fournisseur OAuth2 ne supporte pas le type de réponse 'code'."
+    untestedPowerShellVersionWarningMessage                           = "[AVERTISSEMENT] Pode {0} n'a pas été testé sur PowerShell {1}, car il n'était pas disponible lors de la sortie de Pode."
+    secretVaultAlreadyRegisteredAutoImportExceptionMessage            = "Un coffre-fort secret avec le nom '{0}' a déjà été enregistré lors de l'importation automatique des coffres-forts secrets."
+    schemeRequiresValidScriptBlockExceptionMessage                    = "Le schéma fourni pour le validateur d'authentification '{0}' nécessite un ScriptBlock valide."
+    serverLoopingMessage                                              = 'Boucle du serveur toutes les {0} secondes'
+    certificateThumbprintsNameSupportedOnWindowsExceptionMessage      = 'Les empreintes digitales/Noms de certificat ne sont pris en charge que sous Windows.'
+    sseConnectionNameRequiredExceptionMessage                         = "Un nom de connexion SSE est requis, soit de -Name soit de $`$WebEvent.Sse.Name"
```

Comment:
Typo in the escaping of "$" for "$WebEvent", there's an erroneous "$" in front of the backtick